aws/resource_aws_guardduty_filter_test.go

```diff
@@ -257,7 +257,7 @@ resource "aws_guardduty_filter" "test" {
   finding_criteria {
     criterion {
       field  = "region"
-      equals = ["eu-west-1"]
```

Comment:
Similar note here and below about potentially using `aws_region` data source instead (the check can be updated to `testAccGetRegion()` for simplicity.

Reply:
Completed